### PR TITLE
Fixed #27680 -- Add ccbv.co.uk to documentation.

### PR DESCRIPTION
**Pull request by jpic**
[Ticket](https://code.djangoproject.com/ticket/27680)

### CODE REVIEW

File: docs/topics/class-based-views/intro.txt [hunk 1]
```diff
@@ -173,6 +173,13 @@ that inherits from ``View`` - for example, trying to use a form at the top of a
 list and combining :class:`~django.views.generic.edit.ProcessFormView` and
 :class:`~django.views.generic.list.ListView` - won't work as expected.
 
+.. note::
```
Comment (timgraham) [inline docs/topics/class-based-views/intro.txt]: I'd use `.. seealso::`  or  `.. admonitition::` with a more descriptive title than "Note".

File: docs/topics/class-based-views/intro.txt [hunk 2]
```diff
@@ -173,6 +173,13 @@ that inherits from ``View`` - for example, trying to use a form at the top of a
...
+
+   The `CCBV`_ website provides a user friendly way to navigate
```
Comment (timgraham) [inline docs/topics/class-based-views/intro.txt]:
```
`Classy Class-Based Views <https://ccbv.co.uk/>`_ provides a nice interface to easily navigate
the class hierarchy of the built-in class-based views.
```

**Comment by jpic** [comment]
> I wonder if docs/ref/class-based-views/flattened-index.txt might be a better or additional place to mention that.

I just added it to the flattened-index, I wonder if this should be removed from the intro too as well.

**Comment by timgraham** [comment]
I guess the mention in the index is sufficient. It seems a bit out of place in the intro.
merged in 944610a86c81f90c3c79e5440a5e2c706aa0ef62, thanks!

**Pull request closed without merging**